implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -34,14 +50,43 @@ private static void getDefaultFromClasspath(File propertyFile) {
         }
     }
 
+    private static PropertiesConfiguration getReferenceProperties(File propertyFile) {
+        try {
+            ClassPathResource classPathResource = new ClassPathResource(propertyFile.getName());
+            return new PropertiesConfiguration(classPathResource.getURL());
+        } catch (ConfigurationException | IOException e) {
+            throw new IllegalStateException(e);
+        }
+    }
+
+    private static PropertiesConfiguration getTargetProperties(File propertyFile) {
```

Comment:
By target do you mean properties file in jbb dir or what? 
How about "getTargetPropertiesInJbbDirectory()" ?